stable/aws-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,38 @@
+tlsSecretName: pod-identity-webhook
+annotationPrefix: eks.amazonaws.com
+tokenAudience: sts.amazonaws.com
+
+replicas: 3
+
+image:
+  repository: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pod-identity-webhook
+  tag: latest
```

Comment:
The tag should be a a git SHA or sem ver. With latest, the same replica set could run different "versions" of latest. How would a user suppose to upgrade the chart? Helm will see no changes in the image tag and running `helm upgrade` will not start a rolling update.

Reply:
OK I set it to `0.2.0`, the latest release. It's cosmetic anyway as the docker images don't exist yet.